lib/Hash/MoreUtils.pm

```diff
@@ -126,6 +138,12 @@ hashref.
 
 If no C<MAP> is given, all keys of C<HASHREF> are assumed mapped to themselves.
 
+=head2 C<slice_missing_map> HASHREF[, MAP]
+
+As C<slice_missing> but only checks for missing keys from MAP in C<HASHREF>.
+
+If no C<MAP> is given, C<slice_missing> will be used only on C<HASHREF>.
```

Comment:
1. slice_missing_map
2. checks for missing keys and map to ... from ...
3. returns ... - empty result when MAP is omitted